recipe_scrapers/_utils.py

```diff
@@ -232,6 +232,18 @@ def normalize_string(string):
     )
 
 
+def csv_to_tags(csv, lowercase=False):
+    raw_tags = csv.split(",")
+    seen = set()
+    tags = []
+    for raw_tag in raw_tags:
+        tag = raw_tag.strip()
+        if tag.lower() not in seen:
```

Comment:
My mistake for not spotting this during the previous code-review, but let's also remove empty values here:

```suggestion
        if tag and tag.lower() not in seen:
```